toolparameteriser/testcreator.py

```diff
@@ -6,25 +6,45 @@
 from string import Template
 import xml.etree.ElementTree as ET
 import toolparameteriser.utils
-toolparameteriser.utils.setlogging()
+import subprocess
+# 
 
 class AbstractTester(ABC):
     def __init__(self,config:dict) -> None:
         super().__init__()
         self.tmplfile="" #Must be set by concrete class
         self.Config=config
-        
-        self.jobs_completed_file=os.path.join(self.Config['output']['path'],"jobs_completed.csv")
+
+        toolparameteriser.utils.setlogging(self.Config["debug"])
```

Comment:
Isn't `setlogging()` called at the start after arguments are parsed?

Reply:
removed